DER/DerivativesContracts/Options.rdf

```diff
@@ -60,28 +66,29 @@
 	xmlns:xsd="http://www.w3.org/2001/XMLSchema#">
 	
 	<owl:Ontology rdf:about="https://spec.edmcouncil.org/fibo/ontology/DER/DerivativesContracts/Options/">
-		<rdfs:label xml:lang="en">Options</rdfs:label>
+		<rdfs:label xml:lang="en">Options Ontology</rdfs:label>
```

Comment:
Do we really need to include "Ontology"? We don't include "Class" in the name of every class!